tests/model_serving/model_server/llmd/utils.py

```
@@ -187,3 +193,242 @@ def get_llmd_pods():
+    Args:
+        prometheus: Prometheus instance
+        llmisvc: The LLMInferenceService
+        workload_pods: List of vLLM workload pods
+        expected_requests: Expected total request count to validate
+
+    Raises:
+        AssertionError: If request count doesn't match expected or cache affinity failed
+    """
+    # Wait for Prometheus to scrape metrics
+    LOGGER.info("Waiting 30 seconds for Prometheus to scrape metrics")
+    time.sleep(30)  # noqa: FCN001
```

Ruff (0.14.7)

Unused `noqa` directive (unknown: `FCN001`)

Remove unused `noqa` directive

(RUF100)